src/interest-groups/join.test.js

```diff
@@ -10,120 +10,102 @@ describe('Interest Groups', () => {
 	describe('joinAdInterestGroup', () => {
 		describe('Errors', () => {
 			describe('function parameters', () => {
-				it('should throw an Error when no options are provided', () => {
-					expect(joinAdInterestGroup()).toThrow();
-					expect(joinAdInterestGroup({})).toThrow();
+				it('throw an Error when no parameters are provided', () => {
```

Comment:
if you are removing `should` perhaps use `test` instead of `it` (`it` is an alias of `test`)

Reply:
commit: 660cb21f29b1958ce80a317f52f03a21cab1354e